cmd/registry/config-md.yml

```diff
@@ -0,0 +1,28 @@
+version: 0.1
+log:
+  fields:
+    service: registry
+    environment: development
+storage:
+    delete:
+      enabled: true
+    filesystem:
+        rootdirectory: /var/lib/registry
+http:
+    addr: :5000
+    debug:
+        addr: localhost:5001
+    headers:
+        X-Content-Type-Options: [nosniff]
+redis:
```

Comment:
It might be okay to rip out that subsystem and cache against new metadata system.